secrets-heroku.properties.SAMPLE

```diff
@@ -1,9 +1,10 @@
-app.namespace=https://NAME-OF-YOUR-APP.herokuapp.com
+app.namespace=https://ucsb-cs-las-f20-6pm-c.herokuapp.com
 app.admin.emails=phtcon@ucsb.edu
 app.member.hosted-domain=ucsb.edu
 
-auth0.domain=FILL-IT-IN
-auth0.clientId=FILL-IT-IN
+auth0.domain=6pm-c.us.auth0.com
+auth0.clientId=p4W0HAdIJv8tMcRQ6aqLhw4NaSXd26Xg
```

Comment:
Nooooo!!!!! we don't put secrets in the SAMPLE file that goes into GitHub!  That's the whole point of having a .SAMPLE file that is separate from the real one.

Now these secrets have to be invalidated and we have to generate new ones, doh!